core/src/main/java/hudson/model/User.java

```diff
@@ -149,7 +149,7 @@
      * accesses a /user/arbitraryName URL.
      * <p>
      * Unfortunately this constitutes a CSRF vulnerability, as malicious users can make admins create arbitrary numbers
-     * of ephemeral user records, so the behavior was changed in Jenkins 2.TODO / 2.32.2.
+     * of ephemeral user records, so the behavior was changed in Jenkins 2.44 / 2.32.2.
```

Comment:
Not obvious from blame, the original change was https://github.com/jenkinsci/jenkins/commit/0a11e21d14cb9f5aa1d97e158619db47db4cb971